groovy/postInit/chemistry/elements/ArsenicChain.groovy

```diff
@@ -88,7 +88,7 @@ for (ore in tab_MAsS) {
         .fluidOutputs(fluid(ore.toLowerCase() + "_leach_solution") * 1000)
         .fluidOutputs(fluid('nitric_oxide') * 7000)
         .duration(240)
-        .EUt(Globals.voltAmps[3])
```

Comment:
HV and MV recipe voltages are fine because they can handle more loss, and superconductors will be prevalent by then